### DISCUSSION TITLE
Surround sound support

### DISCUSSION THREAD

**Discussion by Choum28** [sultim-t/xash-rt#91] [Q&A]
Hello All,

Is it possible to activate surround sound (5.1) support with Xash-RT ?
By default it seems to be stereo only.

Old version of HL (Before Mars 2013, afterValve broke it with linux support.) were supporting surround sound,

**Comment by roboboredom** [comment]
Bump.

**Comment by freecodecampster** [comment]
If you have Creative Alchemy on your system, you can add xash3d.exe to it and it will output surround sound if you have a surround destination like SteelSeries Sonar or 5.1 speakers.

**Reply by Choum28** [reply]
I tried with creative alchemy, it will not ouput a correct surround, it just play all sound from front speakers to rear speakers without taking care to sound position.
The Won version have a real surround sound support when EAX is enabled (without it's stereo only).

**Reply by freecodecampster** [reply]
That's odd I am getting positional audio after using Alchemy on Windows 11 with Half Life legacy on Steam using xash3d.exe as the following screenshot shows the channels are not being duplicated, you can see the channels have different levels. I'll test it on an AV receiver to make 100% sure. 

edit: so I tried it with my AV receiver as the speakers are metres apart and it doesn't sound like some simple channel duplication going on, I could hear different sounds from the rear than the front and the centre channel also was outputting sound, but the surround implementation sounds fairly primitive similar to what Dolby Pro Logic II sounds like. Now whether Creative Alchemy is doing some behind the scenes up-mixing from stereo to surround I don't know but there isn't a 1:1 front/rear mix and it's a lot more immersive than simple stereo.

![Half Life EAX 2024-05-19 163311](https://github.com/sultim-t/xash-rt/assets/28160999/a8856c24-8653-41b2-95fa-bf56e867f041)
![xash3d exe Screenshot 2024 05 19 - 16 40 56 48](https://github.com/sultim-t/xash-rt/assets/28160999/eb68fb27-a112-453b-948f-7cbf1c9f1094)
